src/styles/globals/_breakpoints.scss

```diff
@@ -6,5 +6,5 @@ $breakpoints: (
   // tablets
   'lg': 1280px,
   // desktops
-  'xl': 1920px // xl desktops, tvs,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+  'xl': 1920px // xl desktops, tvs,,,,,,,
```

Comment:
i think prettier is maybe adding these? 😂

Reply:
Yeah, too weird.. i just removed the whole comment :D